src/Access/Common/AccessType.h

```diff
@@ -182,6 +182,7 @@ enum class AccessType
     M(SYSTEM, "", GROUP, ALL) /* allows to execute SYSTEM {SHUTDOWN|RELOAD CONFIG|...} */ \
     \
     M(dictGet, "dictHas, dictGetHierarchy, dictIsIn", DICTIONARY, ALL) /* allows to execute functions dictGet(), dictHas(), dictGetHierarchy(), dictIsIn() */\
+    M(displaySecretsInShowSelect, "", GLOBAL, ALL) /* allows to show plaintext secrets in SELECT and SHOW queries. display_secrets_in_show_and_select_query setting must be turned on, otherwise this privilege does nothing  */\
```

Comment:
maybe `ShowAndSelect`?

Reply:
Current privileges don't have "and" word in them, so I was just following the convention. Again, if you don't mind, I suggest leaving as is.